docs/source/releases/latest/doc-expand-w-plugins.yaml

```diff
@@ -0,0 +1,38 @@
+documentation:
+- title: 'Enhanced documentation for output formatters and checkers'
+  description: 'Added comprehensive documentation for output checkers and formatters, including detailed explanations of how they work, their importance in the testing process, and practical examples of their usage.'
+  files:
+    added: # important changes
+    - 'docs/source/concepts/architecture/interfaces/module_based/output-formatters/index.rst'
+    - 'docs/source/releases/latest/doc-expand-w-plugins.yaml'
+    - 'docs/source/tutorials/extending-with-plugins/procflow.rst'
```

Comment:
Is this correct ? Procflow.rst?

Reply:
ohp, no. I added that but decide it should go in its own PR. Fixing now